Cargo.toml

```diff
@@ -94,6 +94,7 @@ opt-level = 1
 
 # Dependencies that should be kept in sync through the whole workspace
 [workspace.dependencies]
+tokio = { version = "1.22.0", features = ["full", "tracing", "test-util"] }
```

Comment:
I don't believe you need to do this, we can leave each crate to turn on the features they want. eg i think we should try to enforce that the test-util feature is only turned on for tests

Reply:
fixed